### PR TITLE
Added link to Liblouis in Braille support section and contributing guidelines to help section

### PR DESCRIPTION
**Pull request by Adriani90**
### Link to issue number:
None
### Summary of the issue:
n/a
### Description of user facing changes
Added Link to Liblouis to be consistent for what has been done in the synth support section
Added link to contributing guidelines to not only encourage to submit issues and requests, but also to encurage active contribution through other activities as well.
### Description of development approach
n/a
### Testing strategy:
Tested that the links show up correctly in the user guide.

### Known issues with pull request:
None

### Code Review Checklist:

- [x] Documentation:
  - Change log entry
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [ ] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [ ] API is compatible with existing add-ons.
- [ ] Security precautions taken.

### CODE REVIEW

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -292,6 +294,7 @@ Telephone support includes local numbers in Australia and the USA.
 The [email user groups https://github.com/nvaccess/nvda/wiki/Connect] are a great source of community help, as are [certified NVDA experts https://certification.nvaccess.org/].
 
 You can make bug reports or feature requests via [GitHub https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/readme.md].
+Feel also free to follow the [contribution guidelines https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/contributing.md] if you need some inspiration and you are interested to actively contribute  to the community.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:297]: I would suggest linking to the main contributing doc, not the dev one

```suggestion
Feel also free to follow the [contribution guidelines https://github.com/nvaccess/nvda/blob/master/.github/CONTRIBUTING.md] if you need some inspiration and you are interested to actively contribute to the community.
```

File: user_docs/en/userGuide.t2t [hunk 2]
```diff
@@ -58,7 +60,7 @@ NVDA is bundled with [eSpeak NG https://github.com/espeak-ng/espeak-ng], a free,
 Information about other speech synthesizers that NVDA supports can be found in the [Supported Speech Synthesizers #SupportedSpeechSynths] section.
 
 ++ Braille support ++[BrailleSupport]
-For users that own a refreshable braille display, NVDA can output its information in braille.
+For users that own a refreshable braille display, NVDA uses the open source braille translator [Liblouis https://github.com/liblouis/liblouis] to output its information in braille.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:63]: I think this is a little misleading, as we use many different technologies to output information in braille.
Also, this should link to the main LibLouis website

```suggestion
For users that own a refreshable braille display, NVDA can output its information in braille.
NVDA uses the open source braille translator [LibLouis https://liblouis.io/] to generate braille sequences from text.
```

File: user_docs/en/userGuide.t2t [hunk 3]
```diff
@@ -292,6 +295,7 @@ Telephone support includes local numbers in Australia and the USA.
 The [email user groups https://github.com/nvaccess/nvda/wiki/Connect] are a great source of community help, as are [certified NVDA experts https://certification.nvaccess.org/].
 
 You can make bug reports or feature requests via [GitHub https://github.com/nvaccess/nvda/blob/master/projectDocs/issues/readme.md].
+Feel also free to follow the [contribution guidelines https://github.com/nvaccess/nvda/blob/master/.github/CONTRIBUTING.md] if you need some inspiration and you are interested to actively contribute to the community.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:298]: I think this could be rephrased a little better but I'd like to ask @Qchristensen to suggest an alternative

Comment (Qchristensen) [reply user_docs/en/userGuide.t2t:298]: Yes I agree.  Perhaps simply:

The [contribution guidelines https://github.com/nvaccess/nvda/blob/master/.github/CONTRIBUTING.md] contain valuable information for contributing to the community.

**Review by Qchristensen — CHANGES_REQUESTED** [review]

**Review by seanbudd — APPROVED** [review]
thanks @Adriani90 !

**Pull request merged**